pkgs/by-name/ta/tacent/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  ninja,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tacent";
+  version = "0.8.18";
+
+  src = fetchFromGitHub {
+    owner = "bluescan";
+    repo = pname;
+    rev = version;
+    hash = "sha256-z8VuJS8OaVw5CeO/udvBEmcURKIy1oWVYUv6Ai8lTI8=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+  ];
+
+  meta = {
+    description = "C++ library providing linear algebra and various utility functions.";
+    longDescription = ''
+      A C++ library implementing linear algebra, text and file IO, UTF-N conversions,
+      containers, image loading/saving, image quantization/filtering, command-line parsing, etc.
+    '';
+    homepage = "https://github.com/bluescan/tacent";
+    license = lib.licenses.isc;
+    maintainers = [ lib.maintainers.PopeRigby ];
+    platforms = [ lib.platforms.linux ];
```

Comment:
Is this lib linux-only?

Reply:
I believe so. It makes no mention of MacOS support.

Reply:
The source code has some macos-related macros.

https://github.com/bluescan/tacent/blob/f351820ce64f47298d93ae02f948e25a589e5095/Modules/Foundation/Src/tPlatform.cpp#L27